### PR TITLE
new loading

### PR DESCRIPTION
**Pull request by skikozou**
add loading text

### CODE REVIEW

File: components/loading.tsx [hunk 1]
```diff
@@ -11,6 +11,8 @@ const variantTexts = {
 	clock: ['🕐 ', '🕑 ', '🕒 ', '🕓 ', '🕔 ', '🕕 ', '🕖 ', '🕗 ', '🕘 ', '🕙 ', '🕚 '],
 	pipe: ['\\', '|', '/', '-'],
 	point: ['⠙', '⠘', '⠰', '⠴', '⠤', '⠦', '⠆', '⠃', '⠋', '⠉'],
+	wave: [':...', '.:..', '..:.', '...:'],
+	wave: [':...', '.:..', '..:.', '...:','..:.', '.:..',':...'],
```
Comment (EdamAme-x) [inline components/loading.tsx]:
```suggestion
	longwave: [':...', '.:..', '..:.', '...:','..:.', '.:..',':...'],
```
I think this is better

**Review by EdamAme-x — COMMENTED** [review]
please check review!

**Review by skikozou — COMMENTED** [review]
fixed

**Comment by skikozou** [comment]
fixed

**Comment by EdamAme-x** [comment]
@skikozou LGTM!

**Pull request merged**